runtime/testing-basilisk/src/lib.rs

```diff
@@ -858,6 +871,20 @@ impl pallet_multisig::Config for Runtime {
 	type WeightInfo = ();
 }
 
+type XYKLiquidityMiningInstance = warehouse_liquidity_mining::Instance1;
+impl warehouse_liquidity_mining::Config<XYKLiquidityMiningInstance> for Runtime {
+	type AssetId = AssetId;
+	type MultiCurrency = Currencies;
+	type PalletId = WarehouseLMPalletId;
+	type MinTotalFarmRewards = MinTotalFarmRewards;
+	type MinPlannedYieldingPeriods = MinPlannedYieldingPeriods;
+	type BlockNumberProvider = cumulus_pallet_parachain_system::RelaychainBlockNumberProvider<Runtime>;
+	type AmmPoolId = AccountId;
+	type MaxFarmEntriesPerDeposit = MaxEntriesPerDeposit;
+	type MaxYieldFarmsPerGlobalFarm = MaxYieldFarmsPerGlobalFarm;
+	type Event = Event;
+}
+
```

Comment:
```suggestion
static_assertions::const_assert_ne!(
	<Runtime as warehouse_liquidity_mining::Config<XYKLiquidityMiningInstance>>::MaxFarmEntriesPerDeposit::get(),
	0
);
```
If you like to ensure the config is correct

Reply:
I would prefer to do `integrity_test` in warehouse